data/modules/scripts/gamestore/init.lua

```diff
@@ -693,7 +693,7 @@ function Player.canBuyOffer(self, offer)
 			end
 		elseif offer.type == GameStore.OfferTypes.OFFER_TYPE_MOUNT then
 			local hasMount = self:hasMount(offer.id)
-			if hasMount == true then
+			if hasMount then
```

Comment:
you can check function direclty.
```suggestion
			if self:hasMount(offer.id) then
```